Assignment2/src/Screens/HomeScreen.tsx

```diff
@@ -147,14 +128,15 @@ export default function HomeScreen(props: HomeScreenProps) {
                                     />
                                 </div>
                             );
-                        })
-                        }
-                        <DisplayModal editedUser={isEditingUser} users={users} CloseModal={closeEditModal} />
+                        })}
+                        <DisplayModal
+                            editedUser={isEditingUser}
+                            users={users}
```

Comment:
Why DisplayModal have Props "users" or "editedUser"
DisplayModal should be independent of user related data

Display Modal should only know when to be visible and 
content can be pass as children or renderprop only 

"Single Responsibility principle"